forge/src/Jackal.sol

```diff
@@ -136,4 +136,17 @@ abstract contract Jackal {
     function postKey(string memory key) public payable {
         postKeyFrom(msg.sender, key);
     }
+
+    function deleteFileTreeFrom(address from, string memory hash_path, string memory account)
+        public
+        payable
+        validAddress
+        hasAllowance(from)
+    {
+        emit DeletedFileTree(from, hash_path, account);
+    }
+
+    function deleteFileTree(string memory hash_path, string memory account) public payable {
```

Comment:
Same goes here, does this need to be payable?

Reply:
will edit as per our discussion